mmv1/products/discoveryengine/DataStore.yaml

```diff
@@ -0,0 +1,168 @@
+    primary_resource_id: 'unstructured_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_website_basic"
+    primary_resource_id: 'website_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+  - !ruby/object:Provider::Terraform::Examples
+    name: "discoveryengine_datastore_advanced_site_search_basic"
+    primary_resource_id: 'advanced_site_search_basic'
+    primary_resource_name:
+      'fmt.Sprintf("tf_test_data_store%s", context["random_suffix"])'
+    vars:
+      data_store_id: "data-store-id"
+parameters:
+  - !ruby/object:Api::Type::String
+    name: 'location'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The geographic location where the data store should reside.
+  - !ruby/object:Api::Type::String
+    name: 'dataStoreId'
+    required: true
+    immutable: true
+    url_param_only: true
+    description: |
+      The unique id of the data store.
+  - !ruby/object:Api::Type::Boolean
+    name: 'createAdvancedSiteSearch'
+    default_value: false
+    url_param_only: true
+    description: |
+      If true, an advanced data store for site search will be created. If the
+      data store is not configured as site search (GENERIC vertical and
+      PUBLIC_WEBSITE contentConfig), this flag will be ignored.
+
+properties:
+  - !ruby/object:Api::Type::String
+    name: 'name'
+    description: |
+      The unique full resource name of the data store. Values are of the format
+      `projects/{project}/locations/{location}/collections/{collection_id}/dataStores/{data_store_id}`.
+      This field must be a UTF-8 encoded string with a length limit of 1024
+      characters.
+    immutable: true
+    output: true
+  - !ruby/object:Api::Type::String
+    name: 'displayName'
+    description: |
+      The display name of the data store. This field must be a UTF-8 encoded
+      string with a length limit of 128 characters.
+    required: true
+  - !ruby/object:Api::Type::Enum
+    name: 'industryVertical'
+    description: |
+      The industry vertical that the data store registers.
+    values:
+      - :GENERIC
+      - :MEDIA
+    immutable: true
+  - !ruby/object:Api::Type::Array
+    name: 'solutionTypes'
+    description: |
+      The solutions that the data store enrolls. Available solutions for each
+      [industry_vertical][]:
+      `MEDIA`: `SOLUTION_TYPE_RECOMMENDATION` and `SOLUTION_TYPE_SEARCH`.
+      `SITE_SEARCH`: `SOLUTION_TYPE_SEARCH` is automatically enrolled. Other
+      solutions cannot be enrolled.
+    item_type: !ruby/object:Api::Type::Enum
+      name: 'solutionType'
+      description: |
+        The type of solution.
+      values:
+        - :SOLUTION_TYPE_UNSPECIFIED
```

Comment:
Generally unspecified options are not needed from the Terraform interface

Reply:
Thanks, I see. Removed the unspecified value.